test_resolwe_runtime_utils.py

```diff
@@ -52,13 +53,17 @@ def test_improper_input(self):
 
 class TestGenSaveFile(ResolweRuntimeUtilsTestCase):
```

Comment:
If you can, add a test that checks the code when the file or references don't exist.